site/content/api/Event-API.md

```diff
@@ -76,4 +81,82 @@ The following sequence diagram illustrates the flow of messages involved in a *B
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+NB: currently there is only one such *well-known* event.
+
+### Lifecycle Notifications
+
+A specific event can signal that a device is `connected` (e.g. to signal that it is ready to receive a command for a specific time interval) 
+or that a device is `disconnected` (e.g to signal that it is not ready to receive a command anymore).
+ 
+*Business Applications* may want to react to such an event by sending a command upstream to the device.
+For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
+is used to send the command to the appropriate protocol adapter instance.
+
+Such an event may be issued by the device itself or by a protocol adapter detecting that a device is connected or disconnected.
+ For further details on which part of a setup issues such an event see (... concept page).
+
+
+*Lifecycle notifications* may be decided to not being durable, but this is left to the setup of the *AMQP network* and is not further specified in the following.
+
+
+***Lifecycle Notification Payload***
+
+The following table provides an overview of the properties a client needs to set for a *lifecycle notification* event.
+
+| Name           | Mandatory | Location                 | Type      | Description |
+| :------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *content-type* | yes       | *properties*             | *symbol*  | Must be set to *application/vnd.eclipse-hono-notification+json* |
+
+The body of the event request MUST consist of a single *AMQP Value* section containing a UTF-8 encoded string representation of a single JSON object having the following members:
+
+| Name                | Mandatory | Type       | Default Value | Description |
+| :-----------------  | :-------: | :--------- | :----------   | :---------- |
+| *tenant-id*         | *yes*     | *string*   |               | The tenant identifier to send the notification for. |
+| *device-id*         | *yes*     | *string*   |               | The device identifier to send the notification for. |
+| *cause*             | *yes*     | *string*   |               | Value must be either `connected` or `disconnected` . |
+| *source*            | *yes*      | *string*   |               | Value can be either the name of a protocol adapter (`hono-mqtt`, `hono-http`, `hono-kura`), if the device connected/disconnected to the appropriate protocol adapter,  or `device`, if the device sent the notfication itself.
+| *connection-timestamp*  | *no*      | *string*   |   null        | The point in time when the event was created (set by the issuer). If not null, the value MUST be an ISO 8601 compliant combined date and time representation in extended format. **NB:** due to network latencies this timestamp will typically be some time in the past already when the event is received by the application. |
```

Comment:
as I stated above, I strongly believe that we should define a separate, specific event for signaling that a device is ready to receive a command